README.md

```diff
@@ -63,6 +63,7 @@ heroku buildpacks:add heroku/ruby
 | PORT                  | Port Puma will listen on, defaults to 3000                              |
 | RAILS_LOG_TO_STDOUT   | Set by Heroku Ruby buildpack, set manually on other platforms if needed |
 | RAILS_MAX_THREADS     | Number of Puma threads, defaults to 5                                   |
+| HOST                  | Your base URI, e.g. https://myapp.herokuapp.com
```

Comment:
I'm trying to keep this list alphabetically sorted, though `GOOGLE_ANALYTICS_ID` was already out of order, so I'm clearly not very successful at it.